src/types.db

```diff
@@ -192,6 +192,27 @@ snr			value:GAUGE:0:U
 spam_check		value:GAUGE:0:U
 spam_score		value:GAUGE:U:U
 spl			value:GAUGE:U:U
+stenographer_aged_files value:COUNTER:0:U 
+stenographer_current_files value:COUNTER:0:U
+stenographer_http_request_query_POST_bytes value:COUNTER:0:U
+stenographer_http_request_query_POST_completed value:COUNTER:0:U
+stenographer_http_request_query_POST_nanos value:COUNTER:0:U
```

Comment:
There is already a type `total_time_in_ms`. I would add a type `total_time_in_ns` and use it for all `…_nanos` metrics:

```
<host>/stenographer-<name>/total_time_in_ns-http_request_query_POST
<host>/stenographer-<name>/total_time_in_ns-index_base_lookup
<host>/stenographer-<name>/total_time_in_ns-index_set_lookup
<host>/stenographer-<name>/total_time_in_ns-indexfile_read
<host>/stenographer-<name>/total_time_in_ns-packet_read
<host>/stenographer-<name>/total_time_in_ns-packet_scan
```